src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,14 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService serv = new UserService();
+        User user = serv.findByEmail(email);
+        if (user != null) {
+            if (user.getPassword().equals(password)) {
+                return true;
+            }
+        }
+        System.out.println("No user with this e-mail is found");
```

Comment:
don't forget to remove sout (system.out.println) from your solution before committing your HW